server/src/graphql/api.ts

```diff
@@ -62,16 +68,6 @@ export const graphqlRoot: Resolvers<Context> = {
         })
         .limit(25)
```

Comment:
I removed this line so that all artwork is returned, since we didn't implement pagination